indigo/indigo/src/main/scala/indigo/shared/AnimationsRegister.scala

```diff
@@ -8,29 +8,32 @@ import indigo.shared.animation.AnimationRef
 import indigo.shared.datatypes.BindingKey
 import indigo.shared.time.GameTime
 
-import scala.collection.mutable
+final class AnimationsRegister:
```

Comment:
Yeah it's a fair question. They were at one point but I found their lifecycle easier to manage when it was an instance. But maybe it would be better as an object.